flink-annotations/src/main/java/org/apache/flink/annotation/docs/Documentation.java

```diff
@@ -59,6 +59,28 @@
 		int position() default Integer.MAX_VALUE;
 	}
 
+	/**
+	 * Annotation used on config option fields to include them in the "Table Options" section.
```

Comment:
This doesn't seem accurate. There's no "Table Options" section, and this option doesn't control whether anything is included in any documentation, but only serves to introduce additional table-related meta data.

Reply:
Thanks, fixed it.